php/libraries/Utility.class.inc

```diff
@@ -160,7 +160,7 @@ class Utility
             $query .= "WHERE Study_site='Y'";
         }
         if (!$DCC) {
-            $query .= " AND Name != 'DCC'";
+            $query .= " AND Name != 'Data Coordinating Center'";
```

Comment:
The convention we adopted in LORIS a while ago (but obviously after this was written..) is `CenterID <> 1`

Reply:
Do you know if this 'convention' is documented somewhere

Reply:
I'm not sure, it was discussed in a LORIS meeting a year or two ago, so it might be in the minutes. You're right though, it should be better documented somewhere.